release_notes/ocp-4-7-release-notes.adoc

```diff
@@ -768,6 +768,40 @@ The current release fixes this issue. Now, when a rollover occurs in the `indexm
+
+* Previously, when you closed the web terminal connection, all the terminal output from that session, which can be useful to you, disappeared. This issue has been fixed, the terminal output is now retained even after the session is closed. (link:https://bugzilla.redhat.com/show_bug.cgi?id=1909067[*BZ#1909067*])
+
+* Technology preview badges were displayed on the Eventing user interface although it had its GA release. The Technology preview badges are now removed. (link:https://bugzilla.redhat.com/show_bug.cgi?id=1894810[*BZ#1894810*])
+
+* Previously, volume mounts for deployments were not preserved if the deployment was edited using the console edit flows. The modified Deployment YAML overwrote or removed the volume mounts in the pod template specification. This issue has been fixed, the *volume mounts* are now preserved even when the deployment is edited using the console edit flows. (link:https://bugzilla.redhat.com/show_bug.cgi?id=1867965[*BZ#1867965*])
+
+* If triggers used Knative service and In Memory Channel as subscriber, the Knative services and resources were not displayed on the *Topology* view. This issue has been fixed now, so that the Knative data model returns proper data, and the Knative resources are displayed on the *Topology* view. (link:https://bugzilla.redhat.com/show_bug.cgi?id=1906683[*BZ#1906683*])
+
+* Previously, in a disconnected environment, the Helm charts were not displayed in the Developer Catalog due to an invalid configuration while fetching code. This issue has been fixed  by ensuring that proxy environment variables are considered and the helm charts are now displayed on the Developer Catalog. (link:https://bugzilla.redhat.com/show_bug.cgi?id=1918748[*BZ#1918748*])
+
+* While running a Pipeline, the log tab of the `TaskRun` resource displayed the string as `undefined` after the command in the output because This was caused due to some edge cases where some internal string operations printed  `undefined` to the log output. This issue has been fixed now, and the pipeline log output does not drop empty lines from the log stream and does not print the string `undefined` any longer. (link:https://bugzilla.redhat.com/show_bug.cgi?id=1915898[*BZ#1915898*])
+
+* Previously, the port drop-down list in the *Add* flow only provided options for exposed ports and did not allow you to specify a custom port. The drop-down list has now been replaced by a typeahead select menu, and now it is possible to specify a custom port while creating the application. (link:https://bugzilla.redhat.com/show_bug.cgi?id=1881881[*BZ#1881881*])
```

Comment:
s/drop-down list/list/
x2

Should "port" be marked up as a GUI element?

Reply:
Done!

Reply:
I'm not so sure about the port as a GUI element thing, so I will discuss with the team and make the necessary updates